bestie/server/xml_result.go

```diff
@@ -220,6 +226,8 @@ func parseStructuredXml(ts *TestSuite) ([]*xmlResult, error) {
 			xr.result = "fail"
 		} else if len(tc.Skipped.Message) > 0 {
 			xr.result = "skip"
+		} else if len(tc.Error.Message) > 0 {
+			xr.result = "error"
```

Comment:
Please update the comment on line 224 to include 'error' in the list of sections mentioned.

Reply:
Updated in latest